### ISSUE TITLE
Plurals not working

### ISSUE THREAD

**Issue by Nailik** [icerockdev/moko-resources#246]
Hi, 
today i tested the Plurals with Android:
`StringDesc.PluralFormatted(MR.plural.my_plurals, quantity, quantity).toString(context)`
and the quantity in range from 0 to 10.

The plurals xml:
```
<?xml version="1.0" encoding="UTF-8" ?>
<resources>
    <plural name="my_plural">
        <item quantity="zero">%d zero</item>
        <item quantity="one">%d one</item>
        <item quantity="two">%d two</item>
        <item quantity="few">%d few</item>
        <item quantity="many">%d many</item>
        <item quantity="other">%d other</item>
    </plural>
</resources>
```

For 0 i get "other" for 1 i get "one" and for all other numbers from 2 - 10 i always get "other"
So zero and two are not working - and i'm not sure when i should get few an when many ...

Hope you can help find my issue.
I'm using V 0.17.2 and also tried 0.17.0 with the same issues.

**Comment by Nailik** [comment]
Okay i probably missunderstood what the quantity stands for. More information on Android Dev page https://developer.android.com/guide/topics/resources/string-resource

**Comment by sven4all** [comment]
Yes I did the same, the example https://github.com/icerockdev/moko-resources#example-4---plural-formatted-string here is misleading and doesn't match the Android behaviour at least.

TL;DR of the Android dev page is that only the plurals that a language 'needs' are used. So in the English language you only have the difference between 1 and all other quantities. So specifying for zero 'No items' doesn't work, if you want to have that behaviour you should implement that yourselves with a if for 0.
